packages/modsecurity/data_stream/auditlog/elasticsearch/ingest_pipeline/nginx-modsec.yml

```
@@ -0,0 +1,143 @@
+          ignore_missing: true
+      - rename:
+          field: json.transaction.response.Content-Length
+          target_field: http.response.bytes
+          ignore_missing: true
+      - foreach:
+          field: json.transaction.messages
+          ignore_missing: true
+          processor:
+            rename:
+              field: _ingest._value.message
+              target_field: modsec.audit.details.msg           
+      - foreach:
+          field: json.transaction.messages
+          ignore_missing: true
+          processor:
+            rename:
+              field: _ingest._value.details.match
+              target_field: modsec.audit.details.match
+      - foreach:
+          field: json.transaction.messages
+          ignore_missing: true
+          processor:
+            rename:
+              field: _ingest._value.details.file
+              target_field: modsec.audit.details.file
+      - foreach:
+          field: json.transaction.messages
+          ignore_missing: true
+          processor:
+            rename:
+              field: _ingest._value.details.ruleId
+              target_field: modsec.audit.details.rule_id
+      - foreach:
+          field: json.transaction.messages
+          ignore_missing: true
+          processor:
+            rename:
+              field: _ingest._value.details.ver
+              target_field: modsec.audit.details.version
+      - foreach:
+          field: json.transaction.messages
+          ignore_missing: true
+          processor:
+            rename:
+              field: _ingest._value.details.severity
+              target_field: modsec.audit.details.severity    
+
+    # user agent and geoip enrich
+      - user_agent:
+          field: json.transaction.request.headers.User-Agent
+          ignore_missing: true
+      - geoip:
+          field: source.ip
+          target_field: source.geo
+          ignore_missing: true
+      - geoip:
+          database_file: GeoLite2-ASN.mmdb
+          field: source.ip
+          target_field: source.as
+          properties:
+          - asn
+          - organization_name
+          ignore_missing: true
+      - rename:
+          field: source.as.asn
+          target_field: source.as.number
+          ignore_missing: true
+      - rename:
+          field: source.as.organization_name
+          target_field: source.as.organization.name
+          ignore_missing: true
+      - set:
+          field: event.kind
+          value: event
+      - append:
+          field: event.category
+          value: web
+      - append:
+          field: event.type
+          value: access
+      - remove:
+          field: event.original
+          if: "ctx?.tags == null || !(ctx.tags.contains('preserve_original_event'))"
+          ignore_failure: true
+          ignore_missing: true
+      - remove:
+          field:
+            - json
+            - _temps
+          ignore_failure: true
+          ignore_missing: true     
```

```suggestion
          ignore_missing: true
```

Please ensure there is a final new line.